app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    final_list = []
+
+    for human in people:
+
+        final_list.append(
+            Person(human["name"], human["age"])
+        )
+
+    for human in people:
+
+        if "wife" in human and human["wife"] is not None:
+            Person.people[human["name"]].wife = \
+                Person.people[human["wife"]]
```

Comment:
Write inline or split into two lines:
1. Assing wife =
2. Assign Person.people[human["name"]].wife = wife

Reply:
The same with husband